tests/test_circular.py

```diff
@@ -55,6 +55,12 @@ def test_max_recursion_depth(dumps):
 
     rl = sys.getrecursionlimit()
     sys.setrecursionlimit(500)
+
+    if hasattr(sys, "pyston_version_info"):
```

Comment:
Maybe put the original line in the else block?

Reply:
Sure thing, I updated it